x-pack/solutions/observability/plugins/observability_onboarding/test/scout/ui/fixtures/page_objects/onboarding_app.ts

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ScoutPage } from '@kbn/scout-oblt';
+
+export class OnboardingApp {
+  constructor(private readonly page: ScoutPage) {}
+
+  async goto() {
+    await this.page.gotoApp('observabilityOnboarding');
+    await this.page
+      .getByText('What do you want to monitor?')
```

Comment:
This is a "weak" locator and historically does't last long, maybe it is possible to define explicit `data-test-subj` for some UI block that uniquely identifies `OnboardingApp` ?